### ISSUE TITLE
Section 3: Clarify sentences beginning "Solutions to these problems..." 

### ISSUE THREAD

**Issue by rdroms** [Abhayakara/draft-tldr-sutld-ps#20]
From Suzanne Woolf
https://www.ietf.org/mail-archive/web/dnsop/current/msg19170.html

Problems associated with Special-Use Domain Names

7. STYLE/GRAMMAR: The two sentences beginning "Solutions to these problems..." are hard to parse. This is partly because they're making a subtle point, but:

OLD:
   "Solutions to
   these problems are out of scope for this document.  Because of that,
   problems with solutions to these problems are also out of scope for
   this document, and will be covered in a separate document."

NEW:
   "Solutions to these problems, including their costs or tradeoffs, are out of scope
   for this document. They will be covered in a separate document."

**Comment by rdroms** [comment]
I know we discussed this text among ourselves.  I still find the original text a little confusing and I think Suz's suggested text is clearer.  But I don't have a strong opinion either way.

**Comment by Abhayakara** [comment]
How about "Solutions to these problems, including their costs or tradeoffs, are out of scope for this document.   They will be covered in a separate document.   New problems that might be created in the process of solving problems described in this document are also out of scope: these problems are expected to be addressed in the process of evaluating potential solutions."

**Comment by rdroms** [comment]
Ted - your text WFM.

**Comment by wkumari** [comment]
.. and for me. I liked the old text, it made me giggle -- but the proposed its clearer...

**Comment by rdroms** [comment]
Updated text...

OLD:

3.  Problems associated with Special-Use Domain Names

   This section presents a list of problems that have been identified
   with respect to the assignment of special-use names.  Solutions to
   these problems are out of scope for this document.  Because of that,
   problems with solutions to these problems are also out of scope for
   this document, and will be covered in a separate document.

NEW:

3.  Problems associated with Special-Use Domain Names

   This section presents a list of problems that have been identified
   with respect to the assignment of special-use names.  Solutions to
   these problems, including their costs or tradeoffs, are out of scope
   for this document.  They will be covered in a separate document.  New
   problems that might be created in the process of solving problems
   described in this document are also out of scope: these problems are
   expected to be addressed in the process of evaluating potential
   solutions.
